src/main/java/core/basesyntax/CalcArea.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface CalcArea {
+    double calcArea();
```

Comment:
don't be greedy for symbols where not necessary
```suggestion
    double calculateArea();
```